filebeat/harvester/source/pipe.go

```diff
@@ -13,3 +13,4 @@ func (p Pipe) Close() error               { return p.File.Close() }
 func (p Pipe) Name() string               { return p.File.Name() }
 func (p Pipe) Stat() (os.FileInfo, error) { return p.File.Stat() }
 func (p Pipe) Continuable() bool          { return false }
+func (p Pipe) HasState() bool             { return false }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Pipe.HasState should have comment or be unexported